Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-alpine
+
+RUN apk add --update --no-cache gcc g++ && pip install dumb-init
```

Comment:
To keep the layers down, you should consider collapsing these two `apk add`s together